apiary.apib

```diff
@@ -754,6 +754,31 @@ Purchase a share of an investment offer on the secondary marketplace
 
     + Body
 
+## Consents [/users/me/consents]
+
+### List of my consents [GET]
+
+Get the list of consents that a user has agreed on.
+
++ Request
+
+    + Headers
+
+            Content-Type: application/json
+            Authorization: Bearer 674e7f35-7aba-4cec-858c-f48861e9dc21
+
++ Response 200
+
+    + Body
+
+        ```json
+        {
+            "smpConsent": {
+                "agreedOn": "2020-01-28T21:47:33.339+01:00"
+            }
+        }
```

Comment:
How does it look when the user hasn't given the consent? Is the response empty?

Reply:
Also, there is an `/investors/me/restrictions` endpoint, which has a `cannotAccessSmp` boolean field. What's that for, and what is the difference?

Reply:
> How does it look when the user hasn't given the consent? Is the response empty?

Yes, empty body response with HTTP status 200

Reply:
> Also, there is an `/investors/me/restrictions` endpoint, which has a `cannotAccessSmp` boolean field. What's that for, and what is the difference?

`cannotAccessSmp` is internally used to allow / deny access to SMP and has nothing to do with the fact whether the user agreed on the consent or not.